### PR TITLE
NCDA Report - Universal Interventions pane

### PR DESCRIPTION
**Pull request by anvmn**
#518

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_1268

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Pages/WellChild/ProgressReport/View.elm [hunk 1]
```diff
@@ -2147,8 +2160,121 @@ viewTargetedInterventionsPane language currentDate child db questionnairesByAgeI
...
+                            List.repeat 25 ""
+                                |> List.indexedMap
+                                    (\index _ ->
+                                        let
+                                            -- This is the date for last day of month
+                                            -- 'index'. For example, for index = 0, this is
+                                            -- the last day, before child turns 1 month old.
+                                            -- We use it to determine if child was
+                                            -- behind on any of vaccines at that month.
+                                            referrenceDate =
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/WellChild/ProgressReport/View.elm]: `referenceDate` ?

Comment (mariano-dagostino) [reply client/src/elm/Pages/WellChild/ProgressReport/View.elm]: Or maybe just lastDayOfMonth

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good. minor comment.

**Pull request merged**